src/Nri/Ui/Switch/V2.elm

```diff
@@ -214,6 +215,7 @@ viewCheckbox config =
     Html.checkbox config.id
         (Just config.selected)
         [ Attributes.id config.id
+        , Attributes.attribute "role" "switch"
```

Comment:
🎨 Could we use [Accessibility.Styled.Role's `switch`](https://package.elm-lang.org/packages/tesk9/accessible-html/latest/Accessibility-Role#switch) helper instead?